nat-lab/tests/telio_test.py

```diff
@@ -5,6 +5,24 @@
 import utils.testing as testing
 
 
+def create_derpserver_config(state: State) -> DerpServer:
```

Comment:
can you also update `test_handle_derp_event` by leaving the existing test logic and basically copy paste same thing, just update output line with new logic

Reply:
Done